src/abstractdataframe/show.jl

```diff
@@ -556,32 +574,77 @@ end
 #' df = DataFrame(A = 1:3, B = ["x", "y", "z"])
 #' showall(df, true)
 function Base.showall(df::AbstractDataFrame,
-                      splitchunks::Bool = false) # -> Void
-    showall(STDOUT, df, splitchunks)
+                      splitchunks::Bool = false,
+                      onechunk::Bool = false) # -> Void
+    showall(STDOUT, df, splitchunks, onechunk)
     return
 end
 
+#' @exported
 #' @description
 #'
 #' Render a summary of the column names, column types and column missingness
 #' count.
 #'
 #' @param io::IO The `io` to be rendered to.
 #' @param df::AbstractDataFrame An AbstractDataFrame.
+#' @param allcols::Bool If `false` (default) only a subset of columns
+#'        fitting on the screen is printed
```

Comment:
MIssing ending dot here and in other docstrings. Could also use a comma after "(default)" here and elsewhere.